packages/core/admin/admin/src/content-manager/pages/EditView/components/FormInputs/Component/Repeatable.tsx

```diff
@@ -216,30 +187,28 @@ const RepeatableComponent = ({
     );
   };
 
-  let errorMessage = formErrors[name];
-
-  if (hasMinError) {
-    errorMessage = {
-      id: getTranslation('components.DynamicZone.missing-components'),
-      defaultMessage:
-        'There {number, plural, =0 {are # missing components} one {is # missing component} other {are # missing components}}',
-      values: { number: missingComponentsValue },
-    };
-  } else if (componentErrorKeys.some((error) => error.split('.').length > 1) && !hasMinError) {
-    errorMessage = {
-      id: getTranslation('components.RepeatableComponent.error-message'),
-      defaultMessage: 'The component(s) contain error(s)',
-    };
-  }
-
-  if (componentValueLength === 0) {
-    return (
-      <ComponentInitializer error={errorMessage} isReadOnly={isReadOnly} onClick={handleClick} />
-    );
+  // let errorMessage = formErrors[name];
+
+  // if (hasMinError) {
+  //   errorMessage = {
+  //     id: getTranslation('components.DynamicZone.missing-components'),
+  //     defaultMessage:
+  //       'There {number, plural, =0 {are # missing components} one {is # missing component} other {are # missing components}}',
+  //     values: { number: missingComponentsValue },
+  //   };
+  // } else if (componentErrorKeys.some((error) => error.split('.').length > 1) && !hasMinError) {
+  //   errorMessage = {
+  //     id: getTranslation('components.RepeatableComponent.error-message'),
+  //     defaultMessage: 'The component(s) contain error(s)',
+  //   };
+  // }
+
```

Comment:
If there's a reason to keep this there should be a comment explaining why

Reply:
I'm gonna keep commented code for now because this is one of many PRs that will re-introduce a lot of things that are broken, before we merge the feature branch we should definitely remove all this noise for sure.